components/staging/staging.js

```diff
@@ -292,8 +293,11 @@ StagingViewModel.prototype.onAltEnter = function(d, e){
     }
     return true;
 };
+StagingViewModel.prototype.toggleWordWrap = function(state) {
+  this.wordWrap(state);
```

Comment:
So the problem with that is that we have two buttons, "default" and "wrap". if we do `this.wordWrap(!this.wordWrap());` then when a person click "wrap" button twice its gets weird... 

I may just change this button and side by side view buttons to a toggle button later...